### ISSUE TITLE
[MODEL] Start using wandb.ai monitoring

### ISSUE THREAD

**Issue by rojberr** [GHOST-Science-Club/tree-classification-irim#25]
**User story:**
**As a [Maintainer],**
**I want [track and monitor my experiments],**
**So that [decide how to improve my model easily].**

**Acceptance Criteria:**
- [ ] Criteria 1: When /src run locally the results are sent to Wandb
- [ ] Criteria 2: The results are seen in Wandb
- [ ] Criteria 3: We identified helpful parameters and send them also to Wandb. They are included next to the graph
- [ ] Criteria 4: Mention it shortly in README.md. Explain how to use it for others.

**Definition of Done:**
- [ ] All acceptance criteria are met.
- [ ] Documentation: Mention it shortly in README.md

**Description:**

It's free for use for personal and academic projects.
May be run locally.
There is some sort of limit like 100 GB.
Alt to tensorboard.

Helps with concise data presentation, saves the settings and env with logs produced.

Include with
```
import wandb
from wandb.keras import WandbCallback


conf dict = {
                      "GPU": GPU, "Machine":MACHINE,
                      "HM GPU":HM_GPU,
                      "NVLINK": NVLINK,
                    }

wandb.init(
                     project="Wandb irim",
                     entity="what_user_made_it"
                     config=conf_dict,
                     name=f'{MACHINE}-{GPU}-{HM_GPU}GPU' # This will be saved as name of run
                   )

... 
model.fit(x_train, y_train, epochs=3, callbacks=[WandbCallback])
...
```

Parameters that may be worth sending to Wandb:
- Learning rate
- F. celu
- Scheduled learning rate
- F. optymalizacji

**Resources:**
https://wandb.ai/site/
https://wandb.ai/ghost-irim/members
https://www.youtube.com/watch?v=LQvRhQwDOm0
